rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2143,14 +2143,14 @@ bool opcode_00b1(const ScriptArguments& args, const ScriptVehicle vehicle, Scrip
 	opcode 00ba
 	@arg gxtEntry GXT entry
 	@arg time Time (ms)
-	@arg arg3 
+	@arg style
 */
-void opcode_00ba(const ScriptArguments& args, const ScriptString gxtEntry, const ScriptInt time, const ScriptInt arg3) {
-	RW_UNIMPLEMENTED_OPCODE(0x00ba);
-	RW_UNUSED(gxtEntry);
-	RW_UNUSED(time);
-	RW_UNUSED(arg3);
-	RW_UNUSED(args);
+void opcode_00ba(const ScriptArguments& args, const ScriptString gxtEntry, const ScriptInt time, const ScriptInt style) {
+	const auto& text = script::gxt(args, gxtEntry);
+	args.getWorld()->state->text.addText<ScreenTextType::Big>(
+				ScreenTextEntry::makeBig(
+					gxtEntry, text, style, time
+					));
```

Comment:
You might want to move the result of the `ScreenTextEntry::makeBig` to a temporary `auto&` variable to make this code more readable. I first thought the alignment was broken.